research/blanks/configs/scratchpad/eval.yaml

```diff
@@ -0,0 +1,27 @@
+parent: research/conditional/train/configs/baselines/gpt/small.yaml
+md5_parent_hash: cbdf4e5992f41713488c25c819086e51
+time: "0-20:10:00"
```

Comment:
20h, but with an additional 10m? Seems like hitting a "snooze" button in the morning.

Reply:
xD

Reply:
Will disappear after merge of blank evaluation xd